test/integration/crcsuite/crcsuite.go

```diff
@@ -406,7 +411,7 @@ func SetConfigPropertyToValueSucceedsOrFails(property string, value string, expe
 	if value == "current bundle" {
 
 		if bundleEmbedded {
-			value = filepath.Join(CRCHome, bundleName)
+			value = filepath.Join(CRCHome, "cache", bundleName)
 		} else {
 			value = bundleName
 		}
```

Comment:
Have you considered putting these 4 lines in a `getBundleName` function since it's now duplicated in 2 places?

Reply:
Now those 4 lines are actually different, so extracting them to a function didn't seem good anymore. Thanks